.github/workflows/update-canary.yml

```diff
@@ -0,0 +1,41 @@
+name: Update canary branch
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
+  workflow_dispatch:
+
+jobs:
+  updateCanary:
+    name: Update the canary branch
+    if: github.repository == 'nodejs/node-v8' || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone node-v8
+        uses: actions/checkout@v2
+        with:
+          path: node-v8
+
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+
+      - name: Install dependencies
+        run: npm install -g node-core-utils@latest
+
+      - name: Cache V8 clone
+        uses: actions/cache@v2
+        with:
+          path: v8
+          key: v8-clone
+
```

Comment:
```suggestion
```

I'm not sure it's worth having this cache. It is only uploaded on the first successful run so with time we will lose the upside of it.
Currently, it reduces the run time by two minutes.